test/test_mps.py

```diff
@@ -8877,8 +8877,8 @@ def test_cpu_indices(self, device="mps"):
 class TestRNNMPS(TestCaseMPS):
     def test_lstm_1(self, device="mps", dtype=torch.float32):
 
-        rnn = nn.LSTM(1, 4, 2, device="cpu")
-        input = torch.randn(2, 3, 1, device="cpu")
+        rnn = nn.LSTM(7, 4, 2, device="cpu")
+        input = torch.randn(2, 3, 7, device="cpu")
         hx = torch.zeros(2, 3, 4, device="cpu")
         cx = torch.zeros(2, 3, 4, device="cpu")
```

Comment:
This test fails when initialized randomly. But I sometimes cannot reproduce the failure.

Reply:
Fails on CI too